Explorer/Assets/DCL/Time/Systems/UpdateTimeSystem.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using DCL.Time.Components;
+using ECS.Abstract;
+
+namespace DCL.Time.Systems
+{
+    [UpdateInGroup(typeof(PresentationSystemGroup))]
+    public partial class UpdateTimeSystem : BaseUnityLoopSystem
+    {
+        public UpdateTimeSystem(World world) : base(world)
+        {
+            World.Create<TimeComponent>();
+        }
+
+        protected override void Update(float t)
+        {
+            UpdateTickQuery(World, t);
+        }
+
+        [Query]
+        private void UpdateTick(
+            [Data] float dt,
+            ref TimeComponent tickComponent)
+        {
+            tickComponent.Time += dt;
```

Comment:
I believe `deltaTime` is capped at a certain value so it's safer just to assign `UnityEngine.Time.time`